### PR TITLE
fix: use eventify internally to avoid extra events

### PR DESCRIPTION
**Pull request by NickColley**
Allows users to require() the findOrAdd, and have events.

### CODE REVIEW

File: lib/find-or-add-with-events.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+var findOrAddOne = require('../helpers/find-or-add-one')
+var findOrAddMany = require('../helpers/find-or-add-many')
+
+module.exports = findOrAdd
+
+/**
+ * tries to find object in local database, otherwise creates new one
+ * with passed properties.
+ *
+ * @param  {String|Object} idOrObject   id or object with `.id` property
```
Comment (gr2m) [inline lib/find-or-add-with-events.js]: we should either add `@param {Object} state` here, or leave out the jsdoc comments altogether. I think I'd go with the latter

**Comment by gr2m** [comment]
Besides the one little comment, this is good to merge

**Comment by NickColley** [comment]
I've removed the JSDoc comment from lib/find-or-add-with-events.js

**Comment by gr2m** [comment]
merged via 07bd9dd (had to rebase on master) :+1:

**Pull request closed without merging**